117.md

```diff
@@ -0,0 +1,121 @@
+NIP-117
+======
+
+Bounties
+------
+
+`draft` `optional` `author:ChristianChiarulli`
+
+## Abstract
+
+This NIP defines a standard for creating, applying to and assigning bounties.
+
+## Event Overview
+
+1. **Bounty Event:** defined as a parameterized replaceable event of kind 30050. This event is published by the creator of the bounty.
+
+2. **Bounty Application Event:** defined as a regular event of kind 8050. This event is published by an applicant and should reference the **Bounty Event**.
+
+## Bounty Event
+
+The `.content` of these events should be a string text in Markdown syntax similar to NIP-23.
+
+The following tags MUST be present:
+
+- A `title` tag to provide a title for the bounty.
+
+- A `reward` tag to provide a reward for the bounty. This is an array in the format `[ "reward", "<number>" ]`,
+
+- A `c` tag to provide the currency of the reward example: "sats" or "usd".
+
+- A `d` tag to provide a unique identifier for the bounty.
+
+- An `s` tag to track the current status of the bounty: `open`, `assigned`, `canceled`.
+
+The following tags MAY be present:
+
+- An `image` tag to show an image along with the bounty.
+
+- A `summary` tag to provide a short summary of the bounty.
+
+- A `expiration` tag to provide a date and time when the bounty should be completed by.
+
+- A `context` tag to provide a link to a website or other resource that provides more information about the bounty.
+
+- An arbitrary amount of `t` tags to provide a list of tags that can be used as categories to filter bounties.
+
+The following tags MUST be present when the bounty is updated to `assigned` status:
+
+- A `p` tag of the applicant public key to track who is currently assigned to the bounty.
+
+- An `e` tag to reference the **Bounty Application Event**.
+
+- An `application` tag with the stringified **Bounty Application Event**.
+
+## Bounty Application Event
+
+The `.content` of these events should be a simple plain text string explaining why the creator should choose the applicant.
+
+The following tags MUST be present:
+
+- A `d` tag to reference the **Bounty Event**.
```

Comment:
Should this be 'a' tag to reference the 30050 Bounty Event?

Reply:
yea it should be, I'll update this